examples/resources/port_action/main.tf

```diff
@@ -88,6 +88,16 @@ resource "port_action" "restart_microservice" {
           default     = "https://example.com"
           pattern     = "^https://.*"
         }
+        service = {
```

Comment:
Could you run `terraform fmt` on this pretty plz?

Reply:
check how it looks locally